test/utils/jwt/src/proposals.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import fs from "fs";
+import * as Keys from "./keys";
+
+export const issuer = "http://Demo-jwt-issuer";
+
+export const issuerProposal = async (
+  host: string,
+  proposalsPath: string,
+): Promise<void> => {
+  // jwt issuer proposal documentation https://microsoft.github.io/CCF/main/build_apps/auth/jwt.html
+  const jwtIssuerProposal = {
+    actions: [
+      {
+        name: "set_jwt_issuer",
```

Comment:
Does this PR have the logic for processing this proposal once its accepted? Or is that already in the repo?

Reply:
there is a make propose-idp command that will use the output of proposals.ts. Also when make start-host-idp runs, it uses the output of this to add the new idp to the sandbox.